src/NHSD.BuyingCatalogue.Solutions.API/Controllers/AdditionalServiceController.cs

```diff
@@ -12,20 +13,23 @@ public sealed class AdditionalServiceController : ControllerBase
     {
         [HttpGet]
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "Cannot be viewed on swagger if static")]
-        public ActionResult<AdditionalServiceResult> Get([FromQuery] string[] solutionIds)
+        public ActionResult<List<AdditionalServiceResult>> Get([FromQuery] string[] solutionIds)
         {
             if (solutionIds is null)
                 return NotFound();
 
-            return new AdditionalServiceResult
+            return new List<AdditionalServiceResult>
             {
-                Id = SetAdditionalServiceId(solutionIds),
-                Name = "Write on Time Additional Service 1",
-                Summary = "Addition to Write on Time",
-                Solution = new AdditionalServiceSolutionResult
+                new AdditionalServiceResult
                 {
-                    Id = "100000-001",
-                    Name = "Write on Time"
+                    AdditionalServiceId = SetAdditionalServiceId(solutionIds),
```

Comment:
I think the function for generating this ID is wrong now, as it's still using all the solution Ids, not just the current one

Reply:
👍